src/components/Participant/Participant.js

```diff
@@ -39,7 +39,7 @@ function Participant({
   const dispatch = useDispatch();
   const videoRef = React.createRef();
   const cssClassName = `Participant ${focus === 'user' ? 'focus' : ''}`;
-  const showVideo = video || isLocalScreen;
+  const showVideo = (video || isLocalScreen) && !focus;
```

Comment:
this is simple enough but I'd like to glue some JSDocs for the Participant props onto it :)